qi-doc/scribblings/field-guide.scrbl

```diff
@@ -328,7 +328,7 @@ But in an idle moment, this clever shortcut may tempt you:
     (~> (3) ((get-f 1)))
   ]
 
-That is, since Qi typically interprets parenthesized expressions as @seclink["Templates_and_Partial_Application"]{partial application templates}, you might expect that this would pass the value @racket[3] to the function resulting from @racket[(get-f 1)]. In fact, that isn't what happens, and an error is raised instead. As there is only one datum within the outer pair of parentheses in @racket[((get-f 1))], the usual interpretation as partial application would not be useful, and could even lead to unexpected behavior (at least, with the current implementation that uses Racket's @racket[curry]). So instead, Qi attempts to interpret the expression as written, that is, as if it were wrapped in @racket[esc]. As a result, it attempts to evaluate @racket[((get-f 1))] and expects to receive a value that can be used as a @tech{flow} here. If, as in the above expression, the function resulting from @racket[(get-f 1)] expects a single argument, this is now an error as it is being invoked with none.
+That is, since Qi typically interprets parenthesized expressions as @seclink["Templates_and_Partial_Application"]{partial application templates}, you might expect that this would pass the value @racket[3] to the function resulting from @racket[(get-f 1)]. In fact, that isn't what happens, and an error is raised instead. As there is only one datum within the outer pair of parentheses in @racket[((get-f 1))], the usual interpretation as partial application would not typically be useful, so Qi opts to treat it as invalid syntax.
```

Comment:
Does this paragraph mean that `(~> (3) (add1))` is a syntax error? There's a single datum inside the parens.

Reply:
(That program is currently a runtime error:
```
add1: arity mismatch;
 the expected number of arguments does not match the given number
  expected: 1
  given: 0
```

Reply:
Yes, it is saying that `(~> (3) (add1))` would be a runtime error (but if #145 were accepted, it would no longer be a runtime error)